scripts/deblur

```diff
@@ -647,6 +650,8 @@ def workflow(seqs_fp, output_dir, pos_ref_fp, pos_ref_db_fp,
         for f in files_to_remove:
             remove(f)
         rmtree(working_dir)
+        if created_split_dir is not None:
```

Comment:
it is not assured that `created_split_dir` exists as a variable at this point. it is assumed to be the case with the current structure of the logic, but this is a bug waiting to happen if the parallel logic ever changes